### PR TITLE
Add explicit casts to quell Windows warnings.

### PR DESCRIPTION
**Pull request by tjcorona**

### CODE REVIEW

File: src/metaArray.cxx [hunk 1]
```diff
@@ -1171,7 +1171,7 @@ M_ReadElements(METAIO_STREAM::ifstream * _fstream, void * _data,
     else
       {
       _fstream->read((char *)_data, readSize);
-      int gc = _fstream->gcount();
+      int gc = static_cast<int>(_fstream->gcount());
```
Comment (mathstuf) [inline src/metaArray.cxx]: Would this be better as having `gc` be a `std::streamsize`?

File: src/metaBlob.cxx [hunk 2]
```diff
@@ -333,7 +333,7 @@ M_Read(void)
     char* _data = new char[readSize];
     m_ReadStream->read((char *)_data, readSize);
 
-    int gc = m_ReadStream->gcount();
+    int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaBlob.cxx]: Same here.

File: src/metaContour.cxx [hunk 3]
```diff
@@ -416,7 +416,7 @@ M_Read(void)
     char* _data = new char[readSize];
     m_ReadStream->read((char *)_data, readSize);
 
-    int gc = m_ReadStream->gcount();
+    int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaContour.cxx]: `std::streamsize`

File: src/metaContour.cxx [hunk 4]
```diff
@@ -618,7 +618,7 @@ M_Read(void)
       char* _data = new char[readSize];
       m_ReadStream->read((char *)_data, readSize);
 
-      int gc = m_ReadStream->gcount();
+      int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaContour.cxx]: `std::streamsize`

File: src/metaDTITube.cxx [hunk 5]
```diff
@@ -469,7 +469,7 @@ M_Read(void)
     char* _data = new char[readSize];
     m_ReadStream->read((char *)_data, readSize);
 
-    int gc = m_ReadStream->gcount();
+    int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaDTITube.cxx]: `std::streamsize`

File: src/metaImage.cxx [hunk 6]
```diff
@@ -1205,7 +1205,7 @@ CanRead(const char *_headerName) const
 
   char* buf = new char[8001];
   inputStream.read(buf,8000);
-  unsigned long fileSize = inputStream.gcount();
+  unsigned long fileSize = static_cast<unsigned long>(inputStream.gcount());
```
Comment (mathstuf) [inline src/metaImage.cxx]: `std::streamsize`

File: src/metaMesh.cxx [hunk 7]
```diff
@@ -446,7 +446,7 @@ M_Read(void)
     char* _data = new char[readSize];
     m_ReadStream->read((char *)_data, readSize);
 
-    int gc = m_ReadStream->gcount();
+    int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaMesh.cxx]: `std::streamsize`

File: src/metaMesh.cxx [hunk 8]
```diff
@@ -625,7 +625,7 @@ M_Read(void)
       char* _data = new char[readSize];
       m_ReadStream->read((char *)_data, readSize);
 
-      int gc = m_ReadStream->gcount();
+      int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaMesh.cxx]: `std::streamsize`

File: src/metaMesh.cxx [hunk 9]
```diff
@@ -747,7 +747,7 @@ M_Read(void)
     char* _data = new char[readSize];
     m_ReadStream->read((char *)_data, readSize);
 
-    int gc = m_ReadStream->gcount();
+    int gc = static_cast<int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaMesh.cxx]: `std::streamsize`

File: src/metaMesh.cxx [hunk 10]
```diff
@@ -880,7 +880,7 @@ M_Read(void)
   char* _data = new char[pointDataSize];
   m_ReadStream->read((char *)_data, pointDataSize);
 
-  unsigned int gc = m_ReadStream->gcount();
+  unsigned int gc = static_cast<unsigned int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaMesh.cxx]: `std::streamsize`

File: src/metaMesh.cxx [hunk 11]
```diff
@@ -1046,7 +1046,7 @@ M_Read(void)
   char* _celldata = new char[cellDataSize];
   m_ReadStream->read((char *)_celldata, cellDataSize);
 
-  unsigned int gcCell = m_ReadStream->gcount();
+  unsigned int gcCell = static_cast<unsigned int>(m_ReadStream->gcount());
```
Comment (mathstuf) [inline src/metaMesh.cxx]: `std::streamsize`

**Comment by tjcorona** [comment]
@thewtex Would you mind reviewing this merge request?

**Comment by thewtex** [comment]
LGTM.  

std::streamsize may have other issues in how the variables are used later.

**Comment by thewtex** [comment]
Thanks for the contribution!

**Pull request merged**